ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/aaa.md

```diff
@@ -210,6 +210,11 @@ aaa authorization commands 10,15 default group tacacs+ local
 | Commands - Console | 0 | start-stop |  -  | True |
 | Exec - Default | - | start-stop | TACACS | - |
 | System - Default | - | start-stop | TACACS | - |
+<<<<<<< HEAD
+| Dot1x - Default  | - | start-stop | RADIUS | - |
+=======
+| Dot1x - Default  | - | start-stop | TACACS | - |
+>>>>>>> afb82c060bd4a520ba1e8c75fc4c327fbd36212c
```

Comment:
This merge conflict diff should be removed

Reply:
Fixed